packages/simcore-sdk/src/simcore_sdk/node_ports_common/data_items_utils.py

```diff
@@ -20,12 +22,9 @@ def create_simcore_file_id(
     return SimcoreS3FileID(f"{clean_path}")
 
 
-_INTERNAL_DIR = Path(tempfile.gettempdir(), "simcorefiles")
-
-
 def create_folder_path(key: str) -> Path:
-    return Path(_INTERNAL_DIR, f"{threading.get_ident()}", key)
+    return _TMP_SIMCOREFILES / f"{uuid4()}" / key
 
 
 def create_file_path(key: str, name: str) -> Path:
-    return Path(_INTERNAL_DIR, f"{threading.get_ident()}", key, name)
+    return _TMP_SIMCOREFILES / f"{uuid4()}" / key / name

```

Comment:
Analogous here ... use instead https://docs.python.org/3/library/tempfile.html#tempfile.mkstemp

Reply:
This does not create the file, just the path to the file, nor does it enforce it's creation.
Form what I can tell your suggestion has a different usage.